services/identity-service/src/kyc/dtos/RegisterUserDTO.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class RegisterUserDTO {
```

Comment:
Why do we have OTP and password within a DTO. These should be handled by the authentication provider. Only consent should be transmitted.